### PR TITLE
Fix gpu and hibernation feature broken issue.

### PR DESCRIPTION
**Pull request by LiliDeng**

### CODE REVIEW

File: lisa/features/hibernation.py [hunk 1]
```diff
@@ -55,5 +51,8 @@ def can_disable(cls) -> bool:
     def _enable_hibernation(cls, *args: Any, **kwargs: Any) -> None:
         raise NotImplementedError()
 
+    def enabled(self) -> bool:
```
Comment (squirrelsc) [inline lisa/features/hibernation.py:54]: Please fix nvme and security_profile as well.

File: lisa/schema.py [hunk 2]
```diff
@@ -353,9 +353,15 @@ def __hash__(self) -> int:
         return hash(self._get_key())
 
     @staticmethod
-    def create(type: str) -> "FeatureSettings":
+    def create(
+        type: str, extended_schemas: Optional[Dict[Any, Any]] = None
+    ) -> "FeatureSettings":
         # If a feature has no setting, it will return the default settings.
-        return FeatureSettings(type=type)
+        if extended_schemas:
```
Comment (squirrelsc) [inline lisa/schema.py:360]: It doesn't need the check, just call like below.

```python
feature = FeatureSettings(type=type, extended_schemas=extended_schemas)
```

File: lisa/features/hibernation.py (hunk 1 above)
Comment (LiliDeng) [reply lisa/features/hibernation.py:54]: fixed

File: lisa/schema.py (hunk 2 above)
Comment (LiliDeng) [reply lisa/schema.py:360]: if remove the check, will encounter below exception.
```
raise UndefinedParameterError(error_message)
dataclasses_json.undefined.UndefinedParameterError: Received input field with same name as catch-all field: 'extended_schemas': 'None'
```

**Review by squirrelsc — APPROVED** [review]

**Pull request merged**